openedx/stanford/common/djangoapps/enrollment/views.py

```diff
@@ -35,23 +35,23 @@ class UpdateEnrollmentView(APIView):
 
     @method_decorator(require_post_params(['email', 'course_id', 'action']))
     @method_decorator(ensure_csrf_cookie_cross_domain)
-    def post(self, request):
+    def post(self, request, course_key):
         """
         Endpoint to update a user enrollment in a course. Requires staff access.
 
         **Example Request**
 
-            POST /api/enrollment/v1/update_status
+            POST /api/enrollment/v1/status/course-v1:foo+bar+foobar
```

Comment:
Is this what it would look like or no? I'm a bit confused about how this URL gets constructed.